src/main/webapp/app/entities/quiz/quiz-group.model.ts

```diff
@@ -0,0 +1,6 @@
+import { BaseEntity } from 'app/shared/model/base-entity';
+
+export class QuizGroup implements BaseEntity {
+    public id: number;
```

Comment:
baseEntity already has an id, I think you can use the one defined there.

Reply:
updated